index.html

```diff
@@ -539,7 +542,7 @@ <h4>XBOX SERIES X CONTROLLER</h4>
           <h2>What Gamers Offer!</h2>
         </div>
         <div class="row">
-          <div class="col-3">
+          <div class="col-md-3">
             <div class="layer-2 text-center">
               <img src="img/free-delivery 1.png" alt="" />
               <h6 class="my-3">Free Shiping</h6>
```

Comment:
Padding left and right is not like the design

Reply:
Done